scripts/build-pyspy.bat

```diff
@@ -0,0 +1,63 @@
+@echo off
+ECHO Y | RMDIR /S py-spy
+
+git clone --depth 1 -b v0.3.10g1 https://github.com/Granulate/py-spy.git && git -C py-spy reset --hard 480deec8b5dde3cd331d1a793106981c1796d172
```

Comment:
The tag & revision are cloned here. Over time they will deviate from those in `pyspy_build.sh`. Dependencies need to be managed in a single site, not duplicated. Let's do what I suggested here:
> store the hash & tag in files: scripts/pyspy_tag and scripts/pyspy_revision. Then the Linux command becomes git clone --depth 1 -b $(cat pyspy_tag) https://github.com/Granulate/py-spy.git && git -C py-spy reset --hard $(cat pyspy_revision) and a similar one in Windows, and we just have to update the 2 files.